services/projects-service/src/services/imageService.ts

```diff
@@ -201,8 +201,7 @@ export const reimportFailedStudies = async (
                                 `UPDATE xnatprojectstatus 
                                     SET lastreimport = now(),
                                         reimportcount = reimportcount +1
-                                    WHERE xnatprojectstatus.projectid=:projectId
-                                    AND xnatprojectstatus.trustid=:trustId`,
+                                    WHERE xnatprojectstatus.xnatprojectid=:projectId`,
```

Comment:
same here lets refer to this replacement as the `xnatProjectId`
```suggestion
                                    WHERE xnatprojectstatus.xnatprojectid=:xnatProjectId`,
```